library/src/containers/AsyncApi/Standalone.tsx

```diff
@@ -0,0 +1,146 @@
+import React, { Component } from 'react';
```

Comment:
This is the old `AsyncAPI` component but without parser. "New" `AsyncAPI` component used this component directly in `render` function to avoid repetitive code.